prisma:info Calling https://accelerate.prisma-data.net/6.16.2/7db912cae740ad9bfd42967bed92eae713c9bc2fad08374e6b6821f8b54c11b0/graphql (n=0)
prisma:error
Invalid `__TURBOPACK__imported__module__$5b$project$5d2f$src$2f$lib$2f$prisma$2e$ts__$5b$app$2d$route$5d$__$28$ecmascript$29$__["prisma"].user.findUnique()` invocation in
C:\Users\vivu_\Downloads\cis-site\.next\server\chunks\[root-of-the-server]__d6367c99._.js:128:166

  125         status: 400
  126     });
  127 }
→ 128 const existing = await __TURBOPACK__imported__module__$5b$project$5d2f$src$2f$lib$2f$prisma$2e$ts__$5b$app$2d$route$5d$__$28$ecmascript$29$__["prisma"].user.findUnique(
This request could not be understood by the server: {"type":"UnknownJsonError","body":{"code":"P6008","message":"Accelerate was not able to connect to your database. The underlying error is: Error in data model: Diagnostics { errors: [DatamodelError { span: Span { start: 105, end: 124, file_id: FileId(0) }, message: \"Error validating datasource `db`: the URL must start with the protocol `file:`.\" }], warnings: [] }"}} (The request id was: 988047a88c87d417)
Error [PrismaClientKnownRequestError]:
Invalid `__TURBOPACK__imported__module__$5b$project$5d2f$src$2f$lib$2f$prisma$2e$ts__$5b$app$2d$route$5d$__$28$ecmascript$29$__["prisma"].user.findUnique()` invocation in
C:\Users\vivu_\Downloads\cis-site\.next\server\chunks\[root-of-the-server]__d6367c99._.js:128:166

  125         status: 400
  126     });
  127 }
→ 128 const existing = await __TURBOPACK__imported__module__$5b$project$5d2f$src$2f$lib$2f$prisma$2e$ts__$5b$app$2d$route$5d$__$28$ecmascript$29$__["prisma"].user.findUnique(
This request could not be understood by the server: {"type":"UnknownJsonError","body":{"code":"P6008","message":"Accelerate was not able to connect to your database. The underlying error is: Error in data model: Diagnostics { errors: [DatamodelError { span: Span { start: 105, end: 124, file_id: FileId(0) }, message: \"Error validating datasource `db`: the URL must start with the protocol `file:`.\" }], warnings: [] }"}} (The request id was: 988047a88c87d417)
    at <unknown> (src\app\api\auth\register\route.ts:34:40)
    at async POST (src\app\api\auth\register\route.ts:34:22)
  32 |     }
  33 |
> 34 |     const existing = await prisma.user.findUnique({ where: { email } });
     |                                        ^
  35 |     if (existing) {
  36 |       return NextResponse.json({ error: "This email is already registered" }, { status: 409 });
  37 |     } {
  code: 'P5000',
  meta: [Object],
  clientVersion: '6.16.2'
}
 POST /api/auth/register 500 in 7508ms